src/lib.rs

```diff
@@ -156,50 +156,67 @@ pub fn uring_builder() -> io_uring::Builder {
     io_uring::IoUring::builder()
 }
 
-/// Builder API to allow starting the runtime and creating the io_uring driver with non-default
-/// parameters.
+/// Builder API that can create and start the `io_uring` runtime with non-default parameters,
+/// while abstracting away the underlying io_uring crate.
 // #[derive(Clone, Default)]
 pub struct Builder {
-    entries: u32,
+    sq_entries: u32,
+    cq_entries: u32,
```

Comment:
Sorry, I can't get behind adding a cq entry here.

Reply:
I added the cq entry because I couldn't read the values in the inner`io_uring::Builder`. They're write-only.  
However, I wanted to be able to enforce the invariants regarding SQ and CQ entries.